api/envoy/data/accesslog/v2/BUILD

```diff
@@ -6,6 +6,7 @@ api_proto_library_internal(
     name = "accesslog",
     srcs = ["accesslog.proto"],
     visibility = [
+        "//envoy/config/filter/network/http_connection_manager/v2:__pkg__",
```

Comment:
Why is this needed?

Reply:
I added this because ResponseFlags wasn't visible in proto.